static-checks/html-links/test.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+import re
+import urllib.request
+
+from lib import util, results
+
+
+# Following links are inaccessible because some websites detect automated web scraping
+# and return 'HTTP Error 403: Forbidden' or just hang indefinitely.
+problematic_urls = [
+    'https://www.cyber.gov.au/acsc/view-all-content/ism',
+    'https://docs-prv.pcisecuritystandards.org/PCI%20DSS/Standard/PCI-DSS-v4_0.pdf'
+]
+
+headers = {}
+headers['User-Agent'] = 'Mozilla/5.0 (X11; Linux x86_64;)'
+url_regex = re.compile(r'href=[\'"]?(http[^\'" ]+)', re.IGNORECASE)
+
+with open(util.get_datastream(), 'r') as ds:
+    ds_content = ds.read()
+
+for url in set(url_regex.findall(ds_content)):
+    if url in problematic_urls:
+        results.report('skip', url, 'inaccessible by automated web scraping')
+        continue
```

Comment:
```suggestion
with open(util.get_datastream(), 'r') as ds:
    ds_content = ds.read()
    all_urls = url_regex.findall(ds.read())
    urls = set(filter(lambda x: x not in problematic_urls, all_links))

for url in urls:
```
what do you think about this? Checking in `for` only valid links

Reply:
Or maybe even better would be to not `filter` the `problematic_urls` and treat them like other links. Instead of maintaining such list in `test.py`, we would have them in waivers with other known issues.